tests/integration/FirehoseIntegrationTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of graze/queue.
+ *
+ * Copyright (c) 2015 Nature Delivered Ltd. <https://www.graze.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license https://github.com/graze/queue/blob/master/LICENSE MIT
+ *
+ * @link    https://github.com/graze/queue
+ */
+
+namespace Graze\Queue;
+
+use Aws\ResultInterface;
+use Aws\Firehose\FirehoseClient;
+use Graze\Queue\Adapter\FirehoseAdapter;
+use Mockery as m;
+use Mockery\MockInterface;
+use PHPUnit_Framework_TestCase as TestCase;
+
+class FirehoseIntegrationTest extends TestCase
+{
+    /** @var string */
+    private $queueName;
+    /** @var FirehoseClient|MockInterface */
+    private $firehoseClient;
+    /** @var Client */
+    private $client;
+
+    public function setUp()
+    {
+        $this->deliveryStreamName = 'delivery_stream_foo';
```

Comment:
`The property deliveryStreamName does not exist. Did you maybe forget to declare it?`

Reply:
Now declared